tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -2732,50 +2732,106 @@ class DoNotLogErrorsEndUpRaising(BaseChecker):
     msgs = {"C4762": (
             "Do not log errors that get raised in an exception block.",
             "do-not-log-raised-errors",
-            "Do not log errors at error or warning level when error is raised in an exception block",
+            "Do not log errors when that error is then raised in an exception block",
```

Comment:
nit: I think the wording from the email thread is more clear to me: 

"do-not-log-raised-errors" is to never log an exception that you re-raise "as-is".